bootstrap/mod/src/main/java/org/geysermc/geyser/platform/mod/world/GeyserModWorldManager.java

```diff
@@ -58,16 +62,19 @@
 
 import java.util.HashMap;
 import java.util.List;
+import java.util.UUID;
 import java.util.concurrent.CompletableFuture;
 import java.util.function.Consumer;
 
 public class GeyserModWorldManager extends GeyserWorldManager {
 
     private static final GsonComponentSerializer GSON_SERIALIZER = GsonComponentSerializer.gson();
     private final MinecraftServer server;
+    private static GeyserImpl geyser;
 
-    public GeyserModWorldManager(MinecraftServer server) {
+    public GeyserModWorldManager(GeyserImpl geyser, MinecraftServer server) {
         this.server = server;
+        GeyserModWorldManager.geyser = geyser;
```

Comment:
at this point we may as well just move `handleBlockPlace` into the mixin and use GeyserImpl#getInstance lol

Reply:
sure, will do that